contract_manager/scripts/check_proposal.ts

```diff
@@ -140,7 +140,7 @@ async function main() {
       if (instruction.governanceAction instanceof EvmExecute) {
         // Note: it only checks for upgrade entropy contracts right now
         console.log(
-          `Verifying EVMExecute Contract on ${instruction.governanceAction.targetChainId}`
+          `Verifying EVMExecute on ${instruction.governanceAction.targetChainId}`
```

Comment:
The previous output for verifying the entropy upgrade proposal was very confusing and made it look like you were upgrading the executor contract. The new output is very explicit:

```
Verifying EVMExecute on blast_s2_testnet
blast_s2_testnet  call upgradeTo(address) with arguments (0x014393b4c3c28054D533204309c52c77373Bea24) on EvmEntropyContract at address:98046bd286715d3b0bc227dd7a956b83d8978603 from executor:5f3c61944ceb01b3eaef861251fb1e0f14b848fb.
blast_s2_testnet    new implementation address:0x014393b4c3c28054D533204309c52c77373Bea24 has code digest:0x00796fddbb5c438e486af88939daaaade9560d659124099113b05c9fb5c82fb1
```